src/main/resources/assets/gregtech/sounds.json

```diff
@@ -397,5 +397,15 @@
         "stream" : false
       }
     ]
+  },
+  "click" : {
+    "subtitle" : "gregtech.subtitle.click",
+    "category" : "block",
```

Comment:
I don't think the block category is correct for this sound, is there an item category that we can put this in?